app/models/menu_item.rb

```diff
@@ -10,6 +10,10 @@ class MenuItem < ApplicationRecord
   scope :visible, -> { where(visible: true).joins(:category).order('name ASC') }
   scope :ungrouped, -> { where.missing(:group_menu_item) }
 
+  def price_dollars
+    price / 100.0
```

Comment:
price is stored in the db as cents, so this method converts it to dollars and cents